### ISSUE TITLE
Delimiter changed inside statement error

### ISSUE THREAD

**Issue by SamuraiPrinciple** [flyway/flyway#2726]
###### Which version and edition of Flyway are you using?

6.3.1

###### If this is not the latest version, can you reproduce the issue with the latest one as well?
(Many bugs are fixed in newer releases and upgrading will often resolve the issue)

###### Which client are you using? (Command-line, Java API, Maven plugin, Gradle plugin)

Command-line

###### Which database are you using (type & version)?

MySQL 8.0.19

###### Which operating system are you using?

macOS, Centos 7

###### What did you do?
(Please include the content causing the issue, any relevant configuration settings, the SQL statement that failed (if relevant) and the command you ran.)

Run `flyway migrate` on the attached project

###### What did you expect to see?

A stored procedure being created.

###### What did you see instead?

```
Flyway Community Edition 6.3.1 by Redgate
Database: jdbc:mysql://localhost (MySQL 8.0)
Successfully dropped pre-schema database level objects (execution time 00:00.001s)
Successfully dropped schema `repro` (execution time 00:00.011s)
Successfully dropped post-schema database level objects (execution time 00:00.001s)
Flyway Community Edition 6.3.1 by Redgate
Database: jdbc:mysql://localhost (MySQL 8.0)
Creating schema `repro` ...
Creating Schema History table `repro`.`flyway_schema_history` ...
Current version of schema `repro`: null
Migrating schema `repro` to version 1.0 - repro
ERROR: Unable to parse statement in sql/R__1_repro.sql at line 4 col 1: Delimiter changed inside statement at line 4 col 1: create procedure proc1()
begin
  update t1 set col1 = if (true, 0, 1);
end$$
```

Also, removing space between *if* and *(* makes the problem go away

[repro.zip](https://github.com/flyway/flyway/files/4343045/repro.zip)

**Comment by Lyeeedar** [comment]
Fixed in 2fa1720d3599a0a9f03c52cc8e4100541b11f08f

**Comment by juliahayward** [comment]
... which will be in the 6.3.2 release
